docs/guides/caching.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Caching in Cortex"
+linkTitle: "Caching in Cortex"
+weight: 1
```

Comment:
I would argue it's not the first item of the menu we want (ie. look better to keep "Running Cortex in Production" as first item):

```suggestion
weight: 5
```